communications/newsletters/newsletter_02_Oct2019.md

```diff
@@ -0,0 +1,87 @@
+
+Thanks to [Paula Andrea Martinez](https://github.com/orchid00), National Characterisation Training Coordinator at the Australian National Imaging Facility, we've already added an "Australia-friendly" cafe on 30 October. :sparkling_heart:
+
+If you live in a country that is not compatible with our [current schedule](https://github.com/alan-turing-institute/the-turing-way/blob/master/project_management/online-collaboration-cafe.md#attending-an-online-collaboration-cafe) and if you would like to join a call in the future, please add your comment in [this GitHub issue](https://github.com/alan-turing-institute/the-turing-way/issues/711).
+
+There's an issue open to collected your ideas ([#711](https://github.com/alan-turing-institute/the-turing-way/issues/711)).
+Thank you in advance!
+
+## Relevant resources
+
+### Open Life Science Mentoring Program
+
+The Turing Way is delighted to collaborate with the [Open Life Science Program](https://openlifesci.org) organised by Bérénice Batut, Yo Yehudi and Malvika Sharan as one of the [Mozilla Open Leader X projects](https://foundation.mozilla.org/en/opportunity/mozilla-open-leaders/). They are developing this mentoring program for early-stage researchers who want to become ambassadors for Open Life Science in their communities. A call for application for mentees and mentors will be announced soon. Stay up to date through their webpage (https://openlifesci.org/).
+
+## Tips & Tricks for new contributors
+
+Are you new to this project and want to participate in the community? You can start by reading the [contribution guideline](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md), where you can get an overview of the project and the different communication channels you can use to participate.
+
+See [this page](https://github.com/alan-turing-institute/the-turing-way) to find out who is in the community and check out this list of ['good first pull' issues](https://github.com/alan-turing-institute/the-turing-way/labels/good%20first%20issue) to get to know this project well.
+
+## Acknowledgements and celebrations!
+
+Shout out to two of our members [Sarah Gibson](https://twitter.com/drsarahlgibson) and [Rachael Ainsworth](https://twitter.com/rachaelevelyn), who represented The Turing Way project at two separate conferences in September.
+
+Sarah, Research Data Scientist at The Alan Turing Institute, delivered a successful workshop on building a [BinderHub](https://binderhub.readthedocs.io/en/latest/) and _The Turing Way_ at the [RSEconUK 2019](https://rse.ac.uk/conf2019/).
+She shared her training materials in the [GitHub repository](https://github.com/alan-turing-institute/the-turing-way/blob/master/workshops/build-a-binderhub/workshop-presentations/zero-to-binderhub.md) and online via Zenodo (DOI: [10.5281/zenodo.3382208](https://doi.org/10.5281/zenodo.3382208)).
```

Comment:
Do we have a good picture of @sgibson91 from RSECon?

Reply:
<blockquote class="twitter-tweet"><p lang="en" dir="ltr">.<a href="https://twitter.com/drsarahlgibson?ref_src=twsrc%5Etfw">@drsarahlgibson</a> running through a brilliant intro at the Build <a href="https://twitter.com/mybinderteam?ref_src=twsrc%5Etfw">@mybinderteam</a> Hub workshop at <a href="https://twitter.com/hashtag/ukrse19?src=hash&amp;ref_src=twsrc%5Etfw">#ukrse19</a>.<br><br>Learned loads at the first one she ran through <a href="https://twitter.com/hashtag/turingway?src=hash&amp;ref_src=twsrc%5Etfw">#turingway</a> and it&#39;s even more informative this time round! 🤩 <a href="https://t.co/hcCfuIaxZ1">pic.twitter.com/hcCfuIaxZ1</a></p>&mdash; annakrystalli (@annakrystalli) <a href="https://twitter.com/annakrystalli/status/1174603229679763457?ref_src=twsrc%5Etfw">September 19, 2019</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>